Minor updates to make the document better ageing (#20)
mirjak/draft-shmoo-online-meeting: draft-ietf-shmoo-online-meeting.md

In 2020, the COVID-19 pandemic forced the IETF to move all its plenary meetings to online-only events.
This document records the experience gained by holding plenary meetings fully online

> and the guidelines that have evolved from this experience. The aim of this RFC is to document rough

Reviewer: ```suggestion
and the guidelines that have evolved from this experience. The aim of this document is to define rough
```

consensus of these guidelines in the sense that the most participants seemed sufficiently satisfied
with the organization of fully online events as done in 2020 and 2021. These guidelines, however, document only one option

> of running fully online meetings. Similar as the IETF has done it for in-person meetings, changes to the organization

Reviewer: ```suggestion
of running fully online meetings. Like the procedure for in-person meetings, changes to the organization
```

of the meetings and the meeting agenda should be experimented with in the process of establishing future meeting guidelines.

> Also, this document does not cover hybrid meetings (meaning meetings that have higher remote participation

Reviewer: ```suggestion
Hybrid meetings (meaning meetings that have higher remote participation than onsite participation)
```

> than onsite participation). Still, some of the experience gained from fully online meetings might also

Reviewer: ```suggestion
are out of scope. However, some of the experience gained from fully online meetings might also
```

While there is a possible trade-off between shorter but more days,
a compact and potentially intense meeting was slightly preferred from the beginning by the community.

> And, different than for in-person meetings, utilizing time during the

Reviewer: ```suggestion
Unlike in-person meetings, utilizing time during the
```

> And, different than for in-person meetings,  also utilizing time during the

Author: ```suggestion
And, different than for in-person meetings, utilizing time during the
```

But if the total number of requested sessions exceeds the capacity of the usual 8 parallel tracks, it
is possible for a fully online meeting to simply use more tracks. This also means, if the number of meeting days is seen as fixed, this decision
is implicitly made by the working group chairs requesting a certain number of sessions and length.

> Respectively, IETF-111 used 9 paralell tracks for some of the session, but also leading to slightly more schedulin conflicts.

Reviewer: ```suggestion
Respectively, IETF-111 used 9 parallel tracks for some of the session, but also leading to slightly more scheduling conflicts.
```

sessions that are of high interest for a large part of community, such as BoFs and
dispatch meetings, and therefore hard to schedule during the main IETF week.

An experiment where the adminstrative plenary was scheduled on Wednesday in the week before the official session schedule was run for IETF-112.

> The evaluation of the respective survey results indicated that this experiment led to reduction in scheduling conflict but also

Author: ```suggestion
The evaluation of the respective survey results indicated that this experiment led to reduction in scheduling conflicts but also
```

> led to drop in plenary attendance, partly due to limited awareness (see [https://www.ietf.org/blog/ietf112-plenary-experiment-evaluation/]).

Author: ```suggestion
led to a slight drop in plenary attendance, partly due to insufficient awareness (see [https://www.ietf.org/blog/ietf112-plenary-experiment-evaluation/]).
```
